### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'charAt')

### ISSUE THREAD

**Issue by robbeman** [verbb/formie#788]
**Description**

On `/admin/formie/settings/form-templates/edit/{id}` the tabs do not work. Clicking a tab throws

```
TypeError: Cannot read properties of undefined (reading 'charAt')
```

It's not something I need, but the error does seem like it's not supposed to be.

**Steps to reproduce**
1. Create a form template
2. Click one of the tabs

<img width="234" alt="image" src="https://user-images.githubusercontent.com/3245659/156752460-d221ac36-88fd-4c0f-ab9e-62dd5d31baf1.png">

### Form settings
Please pick from the options below.

- Multi-page form: n/a
- Submission Method: n/a
- Client-side Validation: n/a
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.9
- Craft version: Craft Pro 3.7.34
- Multi-site: No

**Additional context**

This is probably because no tab was selected on page load?

**Comment by engram-design** [comment]
I can't seem to replicate this, but I know a similar issue was cause by a Craft issue a little while ago. Can you send through the full error as a screenshot, so I can see where the error is coming from? 

I know there's one possible issue that could cause this, and fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.11](https://github.com/verbb/formie/releases/tag/1.5.11)

**Comment by robbeman** [comment]
Sorry for not replying earlier, it was not something I needed (I just noticed it) and so I lost track of it. It is fixed in the latest version of craft and formie. Thanks.
